src/hotspot/share/opto/inlinetypenode.hpp

```diff
@@ -28,6 +28,7 @@
 #include "opto/connode.hpp"
 #include "opto/loopnode.hpp"
 #include "opto/node.hpp"
+#include "opto/matcher.hpp"
```

Comment:
style: include reorder

Reply:
I think we should run [IWYU](https://github.com/include-what-you-use/include-what-you-use ) on JVM code once , WDYT?